src/hooks/useLocalStorage.js

```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+
+// Hook
+export default function useLocalStorage(key, initialValue) {
```

Comment:
Instead of a custom hook you could just use the localstorage library with "npm i localstorage". Then import localstorage in the component where you need it and that's it.
You've probably copy-pasted the hook out of the internet and don't know it very well, which is a bit risky. That's what Jerry told me when I did the same 😄 
It's super easy...If you need help let me know.